yktrace/src/tir.rs

```diff
@@ -188,7 +208,9 @@ impl<'a> TirTrace<'a> {
                 continue;
             }
 
-            let stmt = match &body.blocks[user_bb_idx_usize].term {
+            // Each SIR terminator becomes zero or more TIR statements.
+            let mut term_stmts = Vec::new();
+            match &body.blocks[user_bb_idx_usize].term {
```

Comment:
I *think* this should be simplified to just:

```rust
if let Terminator::Call = &body.blocks[user_bb_idx_usize].term {
    ...
}
```

Reply:
I'd agree if there weren't a `Return` case.

Reply:
Oops!